package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "lint": "prettier -c *.js",
     "lint:fix": "prettier --write *.js",
-    "test": "jest"
-  }
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
+  },
+  "type": "module"
 }
```

Comment:
Could we add a development dependency on `static-server` and include a ["start"](https://docs.npmjs.com/cli/start.html) script that runs it?